gst/tensor_decoder/tensordec.c

```diff
@@ -778,8 +416,7 @@ gst_tensordec_get_property (GObject * object, guint prop_id,
     case PROP_MODE:
       if (self->mode == DECODE_MODE_PLUGIN)
         g_value_set_string (value, self->decoder->modename);
-      else
-        g_value_set_string (value, mode_names[self->mode]);
+      g_value_set_string (value, "");
```

Comment:
Please check this line (always sets empty string)

Reply:
Nice catch!